README.md

```diff
@@ -1,284 +1,126 @@
-# `@cloudflare/next-on-pages`
+# ⚡▲ next-on-pages ▲⚡
```

Comment:
I think continuing to refer to this package as `@cloudflare/next-on-pages` rather than `next-on-pages` will have the advantage of reducing confusion when people try to install this. Wrangler v1 had a recurring problem of people trying to install `wrangler` rather than `@cloudflare/wrangler`.

Reply:
I'd suggest changing it everywhere in this doc.

Reply:
ah ok 😓, I'll update this and the rest of the docs 👍

Reply:
Updated all throughout the docs 🙂👍 

I've also noticed this one:
https://github.com/cloudflare/next-on-pages/blob/cc91567c460e6fdc6915448245876bb5744f34d9/src/index.ts#L569

for clarity I guess we'd also. want to say `@cloudflare/next-on-pages` there as well right?
anyways if ok I'd leave it for another PR as I am trying to keep this documentation only 🙂:+1: